lib/susedistribution.pm

```diff
@@ -238,6 +238,7 @@ sub init_consoles {
                 password => $testapi::password
             });
         $self->add_console('install-shell', 'tty-console', {tty => 2});
+        $self->add_console('installation',  'tty-console', {tty => check_var('VIDEOMODE', 'text') ? 1 : 7});
```

Comment:
Sorry, I am afraid I don't understand what is here to explain. The meaning of the code is the same as for Qemu backend (from which it was copied to svirt backend), it switches TTY #.